packages/html-data/src/find-html-attributes.ts

```diff
@@ -0,0 +1,119 @@
+import { writeFile, mkdir } from "node:fs/promises";
+import { Parser, defaultTreeAdapter } from "parse5";
+import type { Attribute, AttributesByTag } from "./types";
+import { htmlPropsDescriptions } from "./descriptions";
+
+// scrub attributes data from https://html.spec.whatwg.org/multipage/indices.html#attributes-3
```

Comment:
wasn't this available in mdn-data package? or some other package? I know entire HTML spec is in jsons at this point, even the official spec has a repo, scraping that stuff seems fragile